lib/api/jenkinsInfo.js

```diff
@@ -0,0 +1,14 @@
+var authUtil = require("../util/auth")
+
+// Retrieve jenkins server information
```

Comment:
Here the doc is written in single line format (// ...) but in the other files is with block format (/** ... */) I suggest using always the block format.